magento/magento2
# Keep maintenance mode on if it was previously enabled

### Description

When enabling maintenance mode it will get disabled after using one of the following commands:
- `bin\magento module:uninstall`
- `bin\magento setup:backup`
- `bin\magento setup:rollback`
- `bin\magento theme:uninstall`
- `bin\magento deploy:mode:set production`

The changes in this PR will first check or maintenance mode was already enabled. If so, it will skip disabling maintenance mode after executing the command. 

### Fixed Issues

1. magento/magento2#9918: Magento 2 automatically disables maintenance mode after certain actions

### Manual testing scenarios

1. Enable maintenance mode: `bin\magento maintenance:enable`
2. Create a backup: `bin/magento setup:backup --media`
3. Confirm that maintenance mode is still enabled: `bin\magento maintenance:status`

The output will be for example:
```
Maintenance mode already enabled
Media backup is starting...
Media backup filename: 1506422834_filesystem_media.tgz (The archive can be uncompressed with 7-Zip on Windows systems)
Media backup path: /var/www/var/backups/1506422834_filesystem_media.tgz
[SUCCESS]: Media backup completed successfully.
Skipped disabling maintenance mode
```
